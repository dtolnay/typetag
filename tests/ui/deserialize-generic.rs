#[typetag::serde]
pub trait Trait<T> {}

fn main() {}
