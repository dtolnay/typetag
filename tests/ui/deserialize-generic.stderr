error: deserialization of generic traits is not supported yet; use #[typetag::serialize] to generate serialization only
 --> tests/ui/deserialize-generic.rs:2:16
  |
2 | pub trait Trait<T> {}
  |                ^^^
